src/gengowatcher/cli.py

```
@@ -90,13 +117,86 @@ def build_argument_parser() -> argparse.ArgumentParser:
+    return str(configured_debug_url or "")
+
+
+def _load_browser_session_snapshot(
+    args: argparse.Namespace, config: AppConfig
+) -> BrowserSessionSnapshot:
+    from . import main as main_module
+
+    debug_url = _resolve_browser_debug_url(args, config)
+    try:
+        return main_module.fetch_browser_session_snapshot_sync(debug_url=debug_url)
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)